### PR TITLE
Email+ substring

### PR DESCRIPTION
**Pull request by laeaton2001**
Possible implementation to #48.
Don't have much experience with handlebars. For some reason I had to specify the arguments as strings but it works.

### CODE REVIEW

**Comment by nblumhardt** [comment]
Thanks for the PR, Luke! Looks great; a couple of thoughts:

 * I'm not sure about the arguments-as-strings aspect, could be surprising to use - is there any discussion in the Handlebars.net repo about this? Numeric literals might be a missing feature there....
 * Could we call it `substring`, since elsewhere in Seq and .NET this is the function name? `Trim()` is usually associated with leading/trailing whitespace.

Cheers!

**Comment by laeaton2001** [comment]
Yep had a look at the handlebars repo & this looks like the issue: https://github.com/rexm/Handlebars.Net/issues/41

I have pulled the latest NuGet locally (1.9.0) & it fixes the numeric literals & handles white space a lot better but the namespace got changed along the way so I needed to update the references as well.

Happy to push the changes but it makes this PR a lot bigger than I thought it was going to be.

File: src/Seq.App.EmailPlus/HandlebarsHelpers.cs [hunk 1]
```diff
@@ -65,5 +66,58 @@ static object FromDynamic(object o)
...
         }
+
+        static void SubstringHelper(TextWriter output, object context, object[] arguments)
+        {
+            //{{ trimString value "0" "30" }}
+            var value = arguments.FirstOrDefault();
+
+            if (value == null)
+            {
+                output.WriteSafeString("null");
```
Comment (nblumhardt) [inline src/Seq.App.EmailPlus/HandlebarsHelpers.cs]: I know the 'pretty printing' helper substitutes `null` for `"null"`, but I'm not sure we should carry this through to substring; can we consider substrings of null to just be empty strings? What do you think?

Comment (laeaton2001) [reply src/Seq.App.EmailPlus/HandlebarsHelpers.cs]: You're quite right; will update to output a empty string.

**Comment by nblumhardt** [comment]
Made two little tweaks, LGTM, if the tests pass let's :shipit: 👍

**Comment by nblumhardt** [comment]
Thanks for the contribution, Luke!

**Pull request merged**